<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceHMIInitsetPage</class>
 <widget class="QWidget" name="MaintainceHMIInitsetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QPushButton
{
		border-radius:8px;

}</string>
  </property>
  <widget class="QPushButton" name="TC1Btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24px, &quot;微软雅黑&quot;;
color: white;background-color: rgb(0, 128, 128);
border: 2px solid white;
</string>
   </property>
   <property name="text">
    <string>TC1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TC2Btn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>200</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: white;background-color: rgb(0, 128, 128);
border: 2px solid white;
</string>
   </property>
   <property name="text">
    <string>TC2</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundlbl">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>180</y>
     <width>800</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
border: 2px solid  white;
border-radius: 12px;
font:36px ;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>610</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>915</x>
     <y>620</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HMIReboot">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>480</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24px, &quot;微软雅黑&quot;;
color: white;background-color: rgb(0, 128, 128);
border: 2px solid white;
</string>
   </property>
   <property name="text">
    <string>重启HMI</string>
   </property>
  </widget>
  <widget class="QLabel" name="TC1HMIDBVlbl">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font:16px ;
</string>
   </property>
   <property name="text">
    <string>TC1 HMI DBVersion:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC2HMIDBVlbl">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>140</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font:16px ;
</string>
   </property>
   <property name="text">
    <string>TC2 HMI DBVersion:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>backgroundlbl</zorder>
  <zorder>TC1Btn</zorder>
  <zorder>TC2Btn</zorder>
  <zorder>label</zorder>
  <zorder>returnBtn</zorder>
  <zorder>HMIReboot</zorder>
  <zorder>TC1HMIDBVlbl</zorder>
  <zorder>TC2HMIDBVlbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
